benchmark/memory/fibers.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+pid = fork do
+	fibers = 100.times.map do
+		Fiber.new{loop{Fiber.yield}}.resume
```

Comment:
I might have missed something, but doesn't this only run once, while the thread example sleeps forever?

Reply:
All the fibers and threads are still alive at the point where you measure memory usage. Does that seem sufficient?